master/lustrefactory.py

```diff
@@ -24,6 +26,12 @@ def do_step_zfs(step):
 def do_step_installdeps(step):
     return do_step_build(step, 'installdeps')
 
+def hide_if_skipped(results, step):
+    return results == SKIPPED
+
+def hide_except_error(results, step):
```

Comment:
Now it is used, but the use doesn't really relate to the subject of the patch.  I won't hold this pull requiest up on that, but try to avoid that in the future.